docs/releases/v2020.2.1.rst

```diff
@@ -0,0 +1,47 @@
+Gluon 2020.2.1
+===============
+
+Added hardware support
+----------------------
+
+- add support for factory images for TP-Link CPE210 3.20 (`#2080 <https://github.com/freifunk-gluon/gluon/issues/2080>`_)
+
+Bugfixes
+--------
+
+- Fixes handling of enabled mesh_on_lan in site configuration (`#2090 <https://github.com/freifunk-gluon/gluon/issues/2090>`_)
+
+- Fixes build issues with lantiq-xrx200 target by removing unsupported DSL modem packages (`#2087 <https://github.com/freifunk-gluon/gluon/pull/2087>`_)
```

Comment:
Is this needed in v2020.1.x as well?

Reply:
@blocktrron

Reply:
Nope, as v2020.1 does not ship OpenSSL when enabling a Gluon feature.  We should be fine there.